assets/src/scripts/components/hydrograph/index.js

```diff
@@ -169,44 +172,52 @@ const plotMedianPoints = function (elem, {visible, xscale, yscale, medianStatsDa
-                .call(link(plotDataLine, createStructuredSelector({
-                    visible: isVisibleSelector('current'),
-                    lines: lineSegmentsSelector('current'),
-                    xScale: xScaleSelector('current'),
-                    yScale: yScaleSelector,
-                    tsDataKey: () => 'current'
-                })))
-                .call(link(plotDataLine, createStructuredSelector({
-                    visible: isVisibleSelector('compare'),
-                    lines: lineSegmentsSelector('compare'),
-                    xScale: xScaleSelector('compare'),
-                    yScale: yScaleSelector,
-                    tsDataKey: () => 'compare'
-                })))
-                .call(link(plotTooltips, createStructuredSelector({
-                    xScale: xScaleSelector('current'),
-                    yScale: yScaleSelector,
-                    data: pointsSelector('current')
-                })))
-                .call(link(plotMedianPoints, createStructuredSelector({
-                    visible: isVisibleSelector('medianStatistics'),
-                    xscale: xScaleSelector('current'),
-                    yscale: yScaleSelector,
-                    medianStatsData: pointsSelector('medianStatistics')
-                })));
+        .append('svg');
+
+
+    svg.call(link((elem, layout) => elem.attr('viewBox', `0 0 ${layout.width} ${layout.height}`), layoutSelector))
+        .call(link(addSVGAccessibility, createStructuredSelector({
+            title: state => state.title,
+            description: state => state.desc,
+            isInteractive: () => true
+        })))
+        .append('g')
+        .attr('transform', `translate(${MARGIN.left},${MARGIN.top})`)
+        .call(link(appendAxes, axesSelector))
+        .call(link(plotDataLine, createStructuredSelector({
+            visible: isVisibleSelector('current'),
+            lines: lineSegmentsSelector('current'),
+            xScale: xScaleSelector('current'),
+            yScale: yScaleSelector,
+            tsDataKey: () => 'current'
+        })))
+        .call(link(plotDataLine, createStructuredSelector({
+            visible: isVisibleSelector('compare'),
+            lines: lineSegmentsSelector('compare'),
+            xScale: xScaleSelector('compare'),
+            yScale: yScaleSelector,
+            tsDataKey: () => 'compare'
+        })))
+        .call(link(plotTooltips, createStructuredSelector({
+            xScale: xScaleSelector('current'),
+            yScale: yScaleSelector,
+            data: pointsSelector('current')
+        })))
+        .call(link(plotMedianPoints, createStructuredSelector({
+            visible: isVisibleSelector('medianStatistics'),
+            xscale: xScaleSelector('current'),
+            yscale: yScaleSelector,
+            medianStatsData: pointsSelector('medianStatistics')
+        })));
+
+    svg.call(link(plotLegend, createStructuredSelector({
```

Comment:
Could this be chained like the other plot functions?

Reply:
Technically no. It's as more that creating a legend on the svg before the lines were there objected with my graphing sensibilities. :alien: 

Irrespective of that, I can make that change.